Python/cpuinfo.c

```diff
@@ -0,0 +1,556 @@
+#include "pycore_cpuinfo.h"
+
+/* CPUID input and output registers are 32-bit unsigned integers */
+#define CPUID_REG                   uint32_t
```

Comment:
We seem to be over-encapsulating it, we shouldn't do that.

Reply:
Semantic purposes are more important in this case. Note that xgetbv has different semantics. And `eax`, `ebx`, `ecx` and `edx` are generic registers that, when named like this, may confuse a future reader (but I don't want to change their names because it's what they are).